_layouts/default.html

```diff
@@ -4,10 +4,18 @@
 
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, user-scalable=no">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 
 	<title>{% if page.title %}{{ page.title }} · {% endif %}{{ site.title }}</title>
 	<meta name="description" content="{{ site.description }}">
+	<meta name="application-name" content="The Lounge">
```

Comment:
Not sure we want that one, according to https://stackoverflow.com/a/24723106

Reply:
Guess should be removed, yeah